<!--   -->
<launch>

<!--cloud parted parameters -->
	<param name="slope" value="99"/>
	<param name="widthOfRalatedRegion" value="3.0"/>
	<param name="distanceOfDetection" value="10.0"/>
	<param name="radiusOfUnrelatedRegion" value="0.3"/>
	<param name="thresholdOfheight" value="-0.5"/>
	
<!--obstacle cluster parameters -->
	<param name="cluster_search_radius" value="0.21"/>


<!--steer creater parameters -->
	<param name="yawErrToSteerAngle_Kp" value="1.0"/>

	
	<node name="parted" pkg="formular" type="parted" output="screen"/>
	
	<node name="clustered" pkg="formular" type="clustered" output="screen"/>
	
	<node name="steer" pkg="formular" type="steer"/>

</launch>
